devices/flash-zynq7000/flashcfg.h

```diff
@@ -29,22 +29,37 @@
 #define CFI_SIZE_REGION(regSize, regCount) (CFI_SIZE_SECTION(regSize) * (regCount + 1))
 
 
+/* If flash supports 4-byte address space returns 1 otherwise returns 0
+   val - flash size in CFI device size unit                             */
+#define CFI_IS_4BYTES_ADDR(val) ((val > 24) ? 1 : 0)
```

Comment:
Used exactly once in flashcfg.c. No need to add this to the header, IMHO this macro is not needed at all:
`info->addrMode = (info->cfi.chipSize > 24) ? flash_4byteAddr : flash_3byteAddr;`
is clear enough, even without any comment.

Reply:
The initial intention was to use it outside `flashcfg`. Curretnly it is not needed. Applied the suggestion.